platform/ios/PolyPodApp/PolyPod/SettingsView.swift

```diff
@@ -11,7 +11,7 @@ struct SettingsView: View {
                     Image("NavIconBackDark").renderingMode(.original)
                 }),
                 center: AnyView(
-                    Text({
+                    Text({ () -> String in
```

Comment:
This is kinda ugly... What about having an extension on `Section` with `title` as computed property which would contain the below switch? Then here just `activeSection.title` is called.